dev-lang/spidermonkey/spidermonkey-140.2.0.ebuild

```
@@ -0,0 +1,466 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+FIREFOX_PATCHSET="firefox-139-patches-03.tar.xz"
```

Please sync this with the firefox-140esr ebuild,
```suggestion
FIREFOX_PATCHSET="firefox-140esr-patches-01.tar.xz"
```